src/html/generator.ml

```diff
@@ -206,6 +207,26 @@ let text_align = function
 
 let cell_kind = function `Header -> Html.th | `Data -> Html.td
 
+(* Turns an inline into a string, for use as alternative text in
+   images *)
+let rec alt_of_inline (i : Inline.t) =
```

Comment:
I'm happy "forbidding" marked-up content in alt texts (just like block content is forbidden in inline content), but I'd still want to parse the content.

Indeed, it would be very strange to me if `{{image:link}Some {b emphasized} content}` were an image with alt text `"Some {b emphasized"` followed by the text `"content}"`.

We could raise a warning (`Warning: '{b ...}' (bold content) is not allowed in '{{:...} ...}' (external link).`, and have the AST contain just a string, as you suggest. But do you agree that we should render it as it is rendered now (`"Some emphasized content"`, stripping the `{b`) ?